index.d.ts

```diff
@@ -40,14 +42,15 @@ Map object keys and values into a new object.
 @param source - Source object to copy properties from.
 @param mapper - Mapping function.
 
+@param options? - Any options which can be applied to the operation
 @example
 ```
 import mapObject = require('map-obj');
 
 const newObject = mapObject({foo: 'bar'}, (key, value) => [value, key]);
 //=> {bar: 'foo'}
 ```
-*/
+ */
```

Comment:
Don't do unrelated changes.

Reply:
```suggestion
*/
```

Sorry, this was by mistake.